README.md

```diff
@@ -1,2 +1,105 @@
 # SeineSailor
-Help developers navigating through the codebase smoothly
+
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+[![GitHub](https://img.shields.io/github/last-commit/SeineAI/SeineSailor/main?style=flat-square)](https://github.com/SeineAI/SeineSailor/commits/main)
+
+## Overview
+
+SeineSailor is an AI-powered copilot for project managers, designed to assist with code reviews, project discussions,
+and knowledge transfer. It leverages the power of WatsonX's `llama3` and `mixt-8x7B` models to provide intelligent and
+context-aware support throughout the development lifecycle.
+
+## Features
+
+- **PR Review and Summarization**: SeineSailor can be summoned to review pull requests and provide a summary of the
+  changes when mentioned in a PR comment.
+- **Project Discussion Participation**: SeineSailor can actively participate in project discussions and brainstorming
+  sessions when invited using the `@SeineSailor` mention in issues or other discussion forums.
+- **Knowledge Transfer**: SeineSailor maintains a knowledge base of the project's development history and can assist new
+  developers by answering questions about the codebase, architecture, and function implementations.
+- **Customizable Prompts**: Tailor the `system_message`, `summarize`, and other prompts to focus on specific aspects of
+  the review process or change the review objective.
```

Comment:
@SeineSailor add single comment

with:
https://github.com/SeineAI/SeineSailor/blob/e2da1a6fc58a2ae2588b58c5249833f05d19dc03/.github/workflows/test_env.yml#L15-L22
and
https://github.com/SeineAI/SeineSailor/blob/e2da1a6fc58a2ae2588b58c5249833f05d19dc03/.github/workflows/test_env.yml#L30-L39